README.md

```diff
@@ -265,13 +275,21 @@ view the [script options output][script_options] for the latest release.
           cmd: phylum-ci --lockfile requirements-prod.txt
           # Thresholds for the five risk domains may be set at the Phylum project level.
           # They can be set differently for CI environments to "fail the build."
-          # NOTE: The shortened form is used here for brevity, but the long form might be more
-          #       descriptive for future readers. For instance `--vul-threshold` instead of `-u`.
-          cmd: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80
+          cmd: >
```

Comment:
I don't think it matters too much, but the string will now have a trailing newline character. To fix that, you have to add a [block chomping indicator](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator)

```suggestion
          cmd: >-
```